libpod/container_internal.go

```diff
@@ -1502,6 +1496,11 @@ func (c *Container) cleanup(ctx context.Context) error {
 		}
 	}
 
+	if err := c.cleanupOverlayMounts(); err != nil {
+		// If the container can't remove content report the error
+		logrus.Errorf("Failed to cleanup overlay mounts for %s: %v", c.ID(), err)
```

Comment:
Why is this in cleanup too? It should already be called as part of cleanupStorage

Reply:
Moved it to cleanupStorage